src/App.tsx

```diff
@@ -8,7 +7,43 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loader, setLoader] = useState(true);
+  const [select, setSelector] = useState('All');
+  const [input, setInput] = useState('');
+  const [todoId, setTodoId] = useState(0);
```

Comment:
will be awesome to create enum for filter values and use null for id default value